### DISCUSSION TITLE
No UDP/DTLS option

### DISCUSSION THREAD

**Discussion by kurtxiangli** [mitmproxy/mitmproxy#6059] [General]
Hi,

I went through the release note of v9 and found the feature "Raw UDP and DTLS Support" is already introduced. And the "Available Options" on Doc also mentions the following two options are available, "rawudp" and "udp-hosts". However, I always encounter the following errors when trying to have them enabled. By checking the available options with '--help', I don't see these options out there. I did the google search as well as the discussions search, only found the discussion "Where is my local traffic? https://github.com/mitmproxy/mitmproxy/discussions/5723" which seems resolved by WireGuard but not my case. Did I miss something? Thanks.

root@kurt-mint-21:~# ./mitmproxy -k --mode transparent --showhost --rawudp
usage: mitmproxy [options]
mitmproxy: error: unrecognized arguments: --rawudp

root@kurt-mint-21:~# ./mitmproxy -k --mode transparent --showhost --udp-hosts 0.0.0.0
usage: mitmproxy [options]
mitmproxy: error: unrecognized arguments: --udp-hosts 0.0.0.0

./mitmproxy --version
Mitmproxy: 9.0.1 binary
Python: 3.11.0
OpenSSL: OpenSSL 3.0.7 1 Nov 2022
Platform: Linux-5.15.0-56-generic-x86_64-with-glibc2.35
lsb_release -a
No LSB modules are available.
Distributor ID: Linuxmint
Description: Linux Mint 21.1
Release: 21.1
Codename: vera

**Comment by mhils** [comment]
Duplicate of #6057
